tools/clang_tidy/test/clang_tidy_test.dart

```diff
@@ -146,7 +146,7 @@ Future<int> main(List<String> args) async {
       errSink: errBuffer,
     );
     final List<io.File> fileList = await clangTidy.computeChangedFiles();
-    expect(fileList.length, lessThan(300));
+    expect(fileList.length, lessThan(2000));
```

Comment:
I remember you had to increase this before as well. I'd be curious to know what's going on with your checkout that is leading to this.

In the meantime, can we change this back?

Reply:
Ah oops this leaked again. I will set it back. I also found out what the issue is. The clang_tidy tool faults to comparing against the 'origin/master' branch, but 'origin/master' on my machine points to my fork, while 'flutter/master' points to the actual master. My master branch in my fork was pretty far behind. I found out that was also making my git presubmit checks take far longer than they should have.